### PR TITLE
Initial adaptation to the Technically organisation

### PR DESCRIPTION
**Pull request by e1himself**
- Change namespace to `Technically`
- Add auto-tests with Github Actions 
- Tweak README a bit
- Add build status badge

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+on: push
+name: Test
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '8.0' ]
```
Comment (someniatko) [inline .github/workflows/test.yml:9]: @e1himself I think we should test also on 7.4 as long as we support this version

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2021 someniatko
+Copyright (c) 2021 Illya Somov <someniatko@gmail.com>
```
Comment (someniatko) [inline LICENSE:1]: I don't think it's necessary to refer to my full name, I guess the nickname is unique enough

File: README.md [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
-A PSR-11 container, wrapping existing container, but restricting access only to selected list of classes.
+`Technically\WhitelistContainer` is a small PSR-11 container utility, 
+that wraps any PSR-11 container instance, but provides access only to selected list of entries.
 
+Any other entries that are not explicitly whitelisted will be reported as not found.
+
+![Tests](https://github.com/technically-php/whitelist-container/actions/workflows/test.yml/badge.svg)
 
 ## Installation
+
```
Comment (someniatko) [inline README.md:11]: I've intentionally removed newline here, to aid visual corellation of the text to the header.

File: README.md [hunk 4]
```diff
@@ -34,3 +39,7 @@ $whitelistContainer->has(Prohibited::class); // false
 $whitelistContainer->get(Allowed::class); // Allowed instance
 $whitelistContainer->get(Prohibited::class); // @throws NotFoundExceptionInterface
 ```
+
+## Credits
+
+Implemented by [Illya Somov][https://github.com/someniatko].
```
Comment (someniatko) [inline README.md]: please refer to me as `someniatko` instead of full name

**Review by someniatko — CHANGES_REQUESTED** [review]

**Comment by someniatko** [comment]
Thank you for the contribution! I have zero knowledge of GitHub Actions though, will need to learn a bit.

File: README.md (hunk 4 above)
Comment (e1himself) [reply README.md]: Fixed with 15af96288931eff3be45ea2410e89193f9855cc9

File: LICENSE (hunk 2 above)
Comment (e1himself) [reply LICENSE]: Fixed with 15af96288931eff3be45ea2410e89193f9855cc9

File: .github/workflows/test.yml (hunk 1 above)
Comment (e1himself) [reply .github/workflows/test.yml]: Fixed with fbd536115e4727abf5afb32fecff2f9042196a09

File: README.md (hunk 3 above)
Comment (e1himself) [reply README.md:11]: :man_shrugging: 

I believe this is a matter of taste and preference.

I have been following the style of the original Markdown author [John Gruber](https://daringfireball.net/projects/markdown/syntax#header) and I think it does make sense for readability in raw format.

Your style is fine too, but I suggest to keep it consistent throughout the organisation and not spend time on these "[bikeshedding](https://en.wiktionary.org/wiki/bikeshedding)" discussions :)

Comment (someniatko) [reply README.md:11]: @e1himself Your argument of following John Gruber's style is invalid, if you look into the [source](https://daringfireball.net/projects/markdown/syntax.text) of the page you're referring to, you'll see he puts **two** newlines before a header, and one newline after. I instead put one before and none after. I'll gladly accept 2+1 as the style guide. However, putting a single line both before and after a section violates the common sense: related stuff should visually come together.

Comment (e1himself) [reply README.md:11]: > I'll gladly accept 2+1 as the style guide. 

This does make sense. Changing it to 2+1. 

Thanks! :+1:

Comment (e1himself) [reply README.md:12]: 2f66801ca950fce81c74ffae2bf0b720cc76b0da

Comment (e1himself) [reply README.md:12]: JFYI I have also updated other Readmes in the organisation:

- `null-container`
- `array-container`
- `dependency-resolver`

**Review by someniatko — APPROVED** [review]

**Comment by someniatko** [comment]
@e1himself I guess it makes sense to squash these commits into one, WDYT?

**Comment by e1himself** [comment]
> I guess it makes sense to squash these commits into one, WDYT?

idk, I usually never squash :shrug: 

In this case, I guess, it might make sense.

**Pull request merged**